### PR TITLE
Updated Windows support

### PR DESCRIPTION
**Pull request by gdavison**
This PR loses docker testing, but re-enables testing the cookbook on Windows. It also adds Windows 2016 support

### CODE REVIEW

**Comment by TD-4242** [comment]
Without docker support in kitchen we lose the ability for travisci to do automated testing.

```Kitchen::UserError: Vagrant 1.1.0 or higher is not installed. Please download a package from http://www.vagrantup.com/downloads.html.```

maybe we could have a separate .kitche.windows.yml and a rake task to run with it.

**Comment by gdavison** [comment]
What do you think of a Travis-specific `.kitchen.yml` with docker, but Vagrant for local testing? And maybe Appveyor for Windows?

**Comment by TD-4242** [comment]
I've been moving all my testing over to docker as it does all the important integration points for the zabbix agent.  It does leave out non-linux testing thought.  I'm not against either two separate kitchen.yml or an erb templated one that reacts to env vars which I have done for some of our internal cookbooks to speed up light change testing and still do full deployment testing when needed.

**Comment by TD-4242** [comment]
I'll validate the tests here in a bit.

File: .kitchen.windows.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  name: vagrant
+
+platforms:
+  - name: windows-2016
+    driver:
+      box: mwrock/Windows2016
+  - name: windows-2012r2
+    driver_config:
+      communicator: winrm
+      box_url: https://wrock.blob.core.windows.net/vhds/vbox2012r2.box
```
Comment (TD-4242) [inline .kitchen.windows.yml]: I can't seem to resolve this URL.

I tried setting:
```
    driver:
      box: mwrock/Windows2012R2
```
and commenting out the box_url setting and it seems to work.

**Pull request merged**